### DISCUSSION TITLE
Should we add an estimated date for removing `setuptools.dep_util`?

### DISCUSSION THREAD

**Discussion by abravalheri** [pypa/setuptools#4128] [Dev]
Right now we have a warning that is easily to miss out:

https://github.com/pypa/setuptools/blob/v69.0.0/setuptools/dep_util.py#L11

In the past we were criticised because of the lack of visibility (https://github.com/pypa/setuptools/issues/3780), so maybe we should promote this to a `SetuptoolsDeprecationWarning`.

@jaraco does it make sense to add a due date for this warning and increase its visibility?

**Comment by jaraco** [comment]
Yes, I agree. Thanks for catching that. 6 months for a due date seems suitable me. Feel free to tackle it or I'll get to it eventually.

**Comment by jaraco** [comment]
Thanks for addressing in #4132. It seems Github doesn't provide cross-references for discussions.
